snap/frontend/README.md

```diff
@@ -0,0 +1,56 @@
+# Snap Frontend
+
+- [Overview](#overview)
+- [Repository layout](#repository-layout)
+- Instructions
+    - [Installation](#installation)
+    - [Test](#test)
+    - [lint](#lint)
+
+## overview
+
+Snap fronted is a Symbol wallet application build on NextJS framework. It need connect to snap backend and interact with the Symbol blockchain.
```

Comment:
Snap fronted is a Symbol wallet application built on the NextJS framework. It must connect to the Snap backend to interact with the Symbol blockchain.